### ISSUE TITLE
i have a couple issues, otherwise this theme would be perfect. 

### ISSUE THREAD

**Issue by gWOLF3** [tomasiser/vim-code-dark#44]
<img width="835" alt="Screen Shot 2020-01-05 at 11 35 50 PM" src="https://user-images.githubusercontent.com/34167140/71833321-a922e080-307a-11ea-81a2-a2cb485fa519.png">
<img width="275" alt="Screen Shot 2020-01-05 at 11 54 48 PM" src="https://user-images.githubusercontent.com/34167140/71833331-ade79480-307a-11ea-90b7-2b5b84609c78.png">

if i 'set background=dark' it fixes, but then the background becomes a gray color instead of black. how do i fix this? 

<img width="631" alt="Screen Shot 2020-01-06 at 11 50 35 AM" src="https://user-images.githubusercontent.com/34167140/71833428-dcfe0600-307a-11ea-8584-067d80c296d8.png">

**Comment by zQueal** [comment]
> hi EndOfBuffer ctermbg=None

**Comment by Ocisra** [comment]
Hi 
I had the same issue, fixed by adding those line in my .vimrc
```
set t_Co=256
colorscheme codedark
highlight Normal ctermbg=NONE
highlight nonText ctermbg=NONE
```
The order is important

**Comment by tomasiser** [comment]
Hello! Do you think we should put these into the color scheme itself?

**Comment by Ocisra** [comment]
Hi,
It would be good I think.
There might be other groups that I haven't thought of (other than `Normal`, `nonText`, `endOfBuffer`) but those are the principal and it should work out of the box

**Comment by saccarosium** [comment]
@tomasiser I think this can be closed since #102.

**Comment by tomasiser** [comment]
Thank you, I'm closing this and feel free to create a new issue whoever falls into anything similar again!
